src/main/java/com/exasol/adapter/document/properties/DocumentAdapterProperties.java

```diff
@@ -48,9 +48,9 @@ public List<EdmlInput> getMappingDefinition() {
     }
 
     /**
-     * Get MAX_PARALLEL_UDFS property value.
+     * Get {@code MAX_PARALLEL_UDFS} property value.
      *
-     * @return configured maximum number of UDFs that are executed in parallel. default: -1
+     * @return configured maximum number of UDFs that are executed in parallel. default: {@code -1}
```

Comment:
Could the approach of VSCJDBC be beneficial here? 
Compare property [TableCountLimit](https://github.com/exasol/virtual-schema-common-jdbc/blob/main/src/main/java/com/exasol/adapter/properties/TableCountLimit.java) and its evaluation in [exasol.adapter.dialects.AbstractSqlDialect.AbstractSqlDialect()](https://github.com/exasol/virtual-schema-common-jdbc/blob/7f2226e778d5c04abd407600a2101a6fee4f2dc6/src/main/java/com/exasol/adapter/dialects/AbstractSqlDialect.java#L61)

Reply:
Yes, we could do this when we add another property, but until then I would leave it as it is :-)